test/controllers/http_proxies_controller_test.rb

```diff
@@ -0,0 +1,46 @@
+require 'test_helper'
+
+class HttpProxiesControllerTest < ActionController::TestCase
+  let(:model) { FactoryGirl.create(:http_proxy) }
+
+  def test_index
```

Comment:
You could use https://github.com/theforeman/foreman/blob/develop/test/controllers/shared/basic_rest_response_test.rb and replace some of the tests like at https://github.com/theforeman/foreman/blob/develop/test/controllers/architectures_controller_test.rb#L8